app/src/main/java/com/epam/bet/viewmodel/SubscribeViewModel.kt

```diff
@@ -0,0 +1,104 @@
+package com.epam.bet.viewmodel
+
+import android.app.Application
+import android.content.Context
+import android.content.SharedPreferences
+import android.util.Log
+import androidx.lifecycle.AndroidViewModel
+import com.epam.bet.entities.Follower
+import com.epam.bet.extensions.showToast
+import com.google.firebase.firestore.FirebaseFirestore
+import com.firebase.ui.firestore.FirestoreRecyclerOptions
+import com.google.firebase.firestore.Query
+
+class SubscribeViewModel(application: Application) : AndroidViewModel(application) {
+
+    private val db = FirebaseFirestore.getInstance()
+    private val users = db.collection("users")
+    private lateinit var sharedPreferences: SharedPreferences
+
+    fun isItMyEmail(email: String): Boolean {
+        sharedPreferences = getApplication<Application>().applicationContext.getSharedPreferences("BetAppSettings", Context.MODE_PRIVATE)
+        val myEmail = sharedPreferences.getString("email", "none")
```

Comment:
Duplicate code (repeated requests to sharedPreferences) 
Put this in a separate function, please !